libflux/flux-core/src/formatter/tests.rs

```diff
@@ -1211,4 +1211,39 @@ fn preserve_multiline_test() {
     return http.post(headers: headers, url: url, data: body)
 }"#,
     );
+    //Checks that a method with >4 params gets collapsed correctly
+    assert_format(
+        r#"selectWindow = (column="_value", fn, as, every, defaultValue, tables=<-) => {
+    _column = column
+    _as = as
+
+    return tables
+        |> aggregateWindow(every: every, fn: fn, column: _column, createEmpty: true)
+        |> fill(column: _column, value: defaultValue)
+        |> rename(fn: (column) => if column == _column then _as else column)"#,
+        r#"selectWindow = (
+        column="_value",
+        fn,
+        as,
+        every,
+        defaultValue,
+        tables=<-) => {
+    _column = column
+    _as = as
+
+    return tables
+        |> aggregateWindow(every: every, fn: fn, column: _column, createEmpty: true)
```

Comment:
We've defined behavior for when `params > 4` and for when `params < 4`. What do we expect to happen here, where `params == 4`?

Reply:
Well aggregate window has 4 params and remains the same so I think our logic for only separating when params > 4 holds.

Reply:
Great, just checking. You may want to clarify that in the comment on line 1240 (i.e., replace `<` with `<=`)

Reply:
Or just nix the comments altogether.

Reply:
Ok I'll update that